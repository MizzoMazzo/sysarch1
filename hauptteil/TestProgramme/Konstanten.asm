lui $1, 0x1234
ori $1, 0x1234
sw $1, 4($0)
