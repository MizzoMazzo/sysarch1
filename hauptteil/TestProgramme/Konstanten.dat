0x3c011234
0x34211234
0xac010004
