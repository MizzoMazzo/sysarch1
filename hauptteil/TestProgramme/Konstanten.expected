@1 12341234
